runtime/util/vmargs.c

```diff
@@ -629,6 +639,19 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 	return OPTION_OK;
 }
 
+static char * 
+getOptionValue(J9VMInitArgs* j9vm_args, IDATA element, const char *optionName)
+{
+	char *optionValue = NULL;
+	const char *option = optionName;
+
+	if (HAS_MAPPING(j9vm_args, element)) {
+		option = MAPPING_MAPNAME(j9vm_args, element);
+	}
+
+	return j9vm_args->actualVMArgs->options[element].optionString + strlen(option);
```

Comment:
Is check for (NULL == option) missed there? I believe strlen(NULL) will crash, is not it?
getOptionValue() can return NULL in this case and caller might check it to return OPTION_ERROR

Reply:
I added check for `(NULL == option)`.
The current users of `getOptionValue()` don't need to check for NULL return value as we already ensure that `optionName` is not NULL.